### DISCUSSION TITLE
mkdir(): Permissão negada

### DISCUSSION THREAD

**Discussion by 0xWeverton** [RamonSilva20/mapos#1778] [Q&A] [answered]
Após toda configuração inicial do sistema, ao cadastrar dados do Emitente, me deparo com o erro `mkdir(): Permissão negada`, alguém já passou por isso? 
![image](https://user-images.githubusercontent.com/57471470/165325925-918774c9-0263-4de0-af6a-8040aab0c30a.png)

**Comment by jsfachini** [comment]


**Reply by 0xWeverton** [reply]
Não sincronizei, não cheguei a ver nada na documentação de instalação.. vou tentar fazer isso, obrigado!!

**Reply by 0xWeverton** [reply]
Alterei o código fonte pra **não** ter como requisito a imagem de upload, e deu certo, se eu tento fazer o upload da imagem, retorna **permissão negada**

**Reply by jsfachini** [reply]
> Não sincronizei, não cheguei a ver nada na documentação de instalação.. vou tentar fazer isso, obrigado!!

Na real não tem nada a ver com a instalação do mapos em si, mas fiz a sincronização com o Google drive, para manter um bkp, após isso apresentou esse erro.

**Comment by willph** [comment]
Está usando Linux?

Deu permissão correta (chmod) na pasta?

**Reply by 0xWeverton** [reply]
Dei sim, inclusive dei permissão no próprio arquivo que está dando erro, que seria o **Mapos.php**

**Reply by 0xWeverton** [reply]
Eu irei tentar reinstalar as dependencias do composer, pois a dependencia fzaninotto/faker deu erro na hora e forcei uma instalação com outro comando

**Comment by willph** [comment, marked as answer]
Qual permissão dada em assets/upload?

**Reply by 0xWeverton** [reply]
Dei permissão de leitura/escrita em **assets/** e deu certo! obrigadoo
